pkg/controllers/dynakube/monitoredentities/reconciler.go

```diff
@@ -40,6 +40,10 @@ func (r *Reconciler) Reconcile(ctx context.Context) error {
 
 	conditions.SetStatusOutdated(r.dk.Conditions(), MEIDConditionType, "Kubernetes Cluster MEID is outdated in the status")
 
+	if !conditions.IsOptionalScopeAvailable(r.dk, dynatrace.ConditionTypeAPITokenEntitiesRead) {
+		log.Info(dynatrace.TokenScopeEntitiesRead + " optional scope not available, skipping reconciliation")
```

Comment:
does it make sense to continue here?

why not just return?

I get that the `dtClient` should be able to handle it "soonTM" but IMO in this case, the whole reconciler is "turned off" if the scope is not provided

Reply:
you can add `// TODOS`, so it is clearer

like `// TODO: move this check inside the dtClient`, because I am worried it will be forgotten